playbooks/roles/appsembler_reporting/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: make logs directory for appsembler_reporting
+  file:
+    path: "{{ appsembler_reporting_log_dir }}"
+    state: "directory"
+    owner: "{{ edxapp_user }}"
+    group: syslog
+    mode: 0755
+
+- name: Create the reporting scripts directory
+  file:
+    path: "{{ appsembler_reporting_scripts_dir }}"
+    state: directory
+    owner: edxapp
+    group: edxapp    
+
+- name: Create the reporting scripts
+  copy:
+    content: >
+      source {{ edxapp_app_dir}}/edxapp_env;
+      /edx/bin/python.edxapp /edx/bin/manage.edxapp lms --settings=aws_appsembler  \
+        generate_appsembler_report {{ item.key }} --no-delay  \
+        2>&1 | tee -a {{ appsembler_reporting_log_dir }}/{{ item.key }}_report.log
+    dest: "{{ appsembler_reporting_scripts_dir }}/{{ item.key }}_report.sh"
+    owner: edxapp
+    group: edxapp
+    mode: 774
+  with_dict: "{{ appsembler_reporting_jobs }}"  
+
+- name: cron jobs for appsembler_reporting tasks
+  cron:
+    name: "run appsembler_reporting {{ item.key }} job"
+    user: "{{ edxapp_user }}"
+    # Set to "minute hour * * *" for example "0 2 * * *"
+    hour: "{{ item.value.cron_h }}" # Hour of the day
+    minute: "{{ item.value.cron_m }}" # Minute of the hour
+    job: "bash -e {{ appsembler_reporting_scripts_dir }}/{{ item.key }}_report.sh"
+  with_dict: "{{ appsembler_reporting_jobs }}"
```

Comment:
`with_dict` -> `with_items`, after addressing the earlier comment about `appsembler_reporting_jobs`.